### DISCUSSION TITLE
openDTU Platine für CMT / NRF Modul

### DISCUSSION THREAD

**Discussion by DcFamas** [tbnobody/OpenDTU#1016] [General]
##
Ich habe seit geraumer Zeit keine Platinen mehr.

**Comment by brunnwart** [comment]
cool, gleich eine bestellt :-)

**Comment by DeDouby** [comment]
Danke 🌻

Ich geh Mal davon aus, wenn man 2 Platinen möchte, dann 2x spenden?

**Reply by DcFamas** [reply]
Exakt. :)

**Comment by tobiknue** [comment]
cool, gleich gespendet und bestellt. Vielen Dank für Eure tolle Arbeit an dem Projekt!

**Comment by harmstorf** [comment]
Ebenfalls gespendet. Lieben Dank euch! ❤️

**Reply by harmstorf** [reply]
PS: ist hier ein 47 μF 35 V Elko eine gute Wahl?

**Reply by DcFamas** [reply]
Das passt soweit. Für meine Platine habe ich die Größe 5*11 gewählt. Grundsätzlich passen aber wirklich fast alle. Also auch 4*7 oder 8*12 oder 6*11. Die 10*X werden dann zu breit, aber wenn man diese dann flach legt ginge das auch. :)

**Comment by Marc9485** [comment]
Wie hoch soll denn die "Spende" sein?😉

**Reply by DcFamas** [reply]
Von 500 bis 1000€ ist alles möglich.
Spaß beiseite, das kannst du dir aussuchen. :)

**Comment by harmstorf** [comment]
Ich habe gerade die Platine eingebaut und vergessen nachzuschauen, wie die Pins belegt sind. 🙈

@DcFamas magst du gerade mal den Display-Part der json config teilen?

**Comment by DcFamas** [comment]
[pin_mapping.txt](https://github.com/tbnobody/OpenDTU/files/11951919/pin_mapping.txt)

Here we go. :) Einfach wieder in eine *.json unbenennen und das entsprechende Modul unter Hardware auswählen. Sind alle drei drin.

**Comment by mimk97** [comment]
Sehr geil :)
Hab was gespendet. Wie kann ich die Platine nun bestellen?

**Reply by DcFamas** [reply]
Gerne an die oben hinterlegte E-Mail deine Adresse schicken. Danke für deine Unterstützung! :)

**Comment by Poldi8488** [comment]
Versendest Du die Platine (gegen Spende wie im ersten Post erwähnt) auch nach Österreich?

**Reply by DcFamas** [reply]
Sofern das nicht überhand nimmt, kriegen wir das auch hin. :)
Schreib mir einfach deine Adresse via. Mail.
Viele Grüße
